test/unit/view.js

```diff
@@ -601,6 +601,242 @@ describe('View class', () => {
+          });
+
+          it('calls getPropertyValue() twice if height is set to 0px in style', () => {
+            view.wrapper.style.height = '0px';
+            assert.equal(view.outerHeight, '0px');
+            assert.calledTwice(getPropertyValueStub);
+            assert.calledWith(getPropertyValueStub, 'height');
+          });
+
+          it('calls getPropertyValue() twice if height is set to auto in style', () => {
+            view.wrapper.style.height = 'auto';
+            assert.equal(view.outerHeight, 'auto');
+            assert.calledTwice(getPropertyValueStub);
+            assert.calledWith(getPropertyValueStub, 'height');
+          });
+
+          it('calls getPropertyValue() once if height is set to a value in style', () => {
+            view.wrapper.style.height = '30px';
+            assert.equal(view.outerHeight, '30px');
+            assert.calledOnce(getPropertyValueStub);
+            assert.calledWith(getPropertyValueStub, 'height');
+          });
+        });
+      });
+
+      describe('className', () => {
+        it('returns an empty string', () => {
+          assert.equal(view.className, '');
+        });
+      });
+
+      describe('shouldResizeX', () => {
+        it('returns false', () => {
+          assert.equal(view.shouldResizeX, false);
+        });
+      });
+
+      describe('shouldResizeY', () => {
+        it('returns false', () => {
+          assert.equal(view.shouldResizeY, false);
+        });
+      });
+
+      describe('document', () => {
+        it('returns iframe\'s document if iframe exists', () => {
+          view.iframe = {document: {}};
+          assert.equal(view.document, view.iframe.document);
+        });
+
+        it('returns window\'s document if iframe does not exist', () => {
+          view.iframe = null;
+          assert.equal(view.document, window.document);
+        });
+      });
+    });
+
+    describe('"private" methods', () => {
+      describe('_createWrapper()', () => {
+        let createElementStub;
+        let appendStub;
+        let mockWrapper;
+
+        beforeEach(() => {
+          mockWrapper = document.createElement('div');
+          createElementStub = sinon.stub(document, 'createElement').returns(mockWrapper);
+          appendStub = sinon.stub(view, 'append');
+        });
+
+        afterEach(() => {
+          createElementStub.restore();
+          appendStub.restore();
+        });
+
+        it('creates a wrapper and appends it to view', () => {
+          view._createWrapper();
+          assert.calledOnce(createElementStub);
+          assert.calledWith(createElementStub, 'div');
+          assert.calledOnce(appendStub);
+          assert.calledWith(appendStub, mockWrapper);
+        });
+
+        it('returns the newly created wrapper', () => {
+          assert.equal(view._createWrapper(), mockWrapper);
+        });
+
+        it('adds component type class to wrapper', () => {
+          component.typeKey = 'typeKey';
```

Comment:
Can we use a more "real" typeKey value? Using `typeKey` here is a bit confusing...